acceptance_tests/features/steps/edit_respondent_details.py

```diff
@@ -104,8 +109,8 @@ def error_email_already_in_use(_):
     assert reporting_unit.save_email_error()
 
 
-def create_respondent(email):
+def create_respondent(email, wait=False):
     email_in_use = get_party_by_email(email)
     if not email_in_use:
         register_respondent(survey_id='cb8accda-6118-4d3b-85a3-149e28960c54', period='201801',
-                            username=email, ru_ref=49900000001)
+                            username=email, ru_ref=49900000001, wait_for_case=wait)
```

Comment:
Shouldn't be logging the email

Reply:
Doesn't really matter for a test suite does it?

Reply:
Shouldn't get into the habit of it, these things have been found to be logged out in prod recently we shouldn't get used to seeing this kind of info in logs

Reply:
These are all test emails, and logging them here is useful as this is a point where things can go wrong and it's helpful to know which respondent has failed to register

Reply:
Apologies I thought I saw it say logger here.